app/pages/stacking/delegated-stacking/components/duration-select-item.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import { Box, Flex, Text } from '@blockstack/ui';
+
+interface DurationSelectItemProps {
+  title: string;
+  durationType: -1 | 1;
```

Comment:
We could use a string literal instead to make this more readable? `'limited' | 'indefinite'`